hyperparams/ppo2.yml

```diff
@@ -263,3 +263,33 @@ InvertedPendulumSwingupBulletEnv-v0:
   ent_coef: 0.0
   learning_rate: 2.5e-4
   cliprange: 0.2
+
+# following https://github.com/lcswillems/rl-starter-files
+# https://stable-baselines.readthedocs.io/en/master/modules/ppo2.html
+MiniGrid-DoorKey-5x5-v0:
```

Comment:
btw, are those parameters working? If you could you add a trained agent to the repo?

Reply:
It is working as embedding everything into a vector for MLP.  Not super good compared to the one you mentioned, though.  CNN for partial map, Embedding/GRU for mission, MLPs for actor/critic should be the way to go...

Reply:
If it is ok with naive models, they are ready.   :->